src/Header/Account/Account.jsx

```diff
@@ -82,24 +83,50 @@ export default class Account extends Component {
 	}
 
 	componentDidMount() {
-		checkIsLoggedIn(this.props.environment)
-			.then(
-				function(data) {
-					if (this.props.onLoginStatusChecked) {
-						this.props.onLoginStatusChecked(data);
-					}
-				}.bind(this)
-			)
-			.catch(
-				function(e) {
-					console.warn("Couldn't load account data", e);
-				}.bind(this)
-			);
+		if (this.state.useIdAM) {
+			//nice accounts supplies links like: {"John Holland":"https://accounts.nice.org.uk/users/143980/editprofile","Sign out":"https://accounts.nice.org.uk/signout"}
+			//idam supplies links like:[{ key: "My profile", value: "/Account/todo" },{ key: "Sign out", value: "/Account/Logout" }]
+			//the following just converts the idam format to the nice accounts format.
+			let links = {};
+			this.props.links.forEach(function(link) {
+				links[link["key"]] = link["value"];
+			});
+			var convertedData = {
```

Comment:
Can you be consistent with let's and var's? Also, these are probably all const's anyway!